src/visGeometry/GeometryStore.ts

```diff
@@ -466,7 +474,8 @@ class GeometryStore {
                 // on updatescene, add instances
                 // on render, pass the group of marchingcubes objects
                 geometry = {
-                    mesh: null,
+                    // null is only acceptable for SphereGroup
+                    mesh: null as unknown as Object3D,
```

Comment:
To clarify: the comment is explaining that it's preferable to force-cast `null` to `Object3D` rather than update this type to allow `null` because `null` should be disallowed for all but one variant of this type?

Reply:
Yes exactly- in fact it might be allowable to just create a `new Object3D()` here, which would be much cleaner, but I didn't try to confirm it.